packages/core/docs/commercetools/composables/use-cart.md

```diff
@@ -87,7 +87,9 @@ type ProductVariant = {
   attributeList: Array<Attribute>;
 }
 
-type CustomQuery = Record<string, string>
+type CustomQuery = {
+  updateCart: string
+}
```

Comment:
I don't think it's the case for both `load` and `save`. Also `load` still has `customQuery?: any`

Reply:
Yup, `load` still has `customQuery?: any` but inside documentation I changed it to `customQuery?: customQuery`